FrontEnd/src/components/CompanyCard/CompanyCard.jsx

```diff
@@ -0,0 +1,209 @@
+                size="medium"
+                count={category.name.toUpperCase()}
+                style={{
+                  backgroundColor: '#1F9A7C',
+                  fontWeight: 600,
+                  fontFamily: 'Inter',
+                  fontSize: 10,
+                  margin: 5,
+                }}
+              />
+            ))
+          : ''}
+      </div>
+    );
+  };
+
+  return (
+    <div className={styles['company-card']}>
+      <div className={styles['company-card__block']}>
+        <div className={styles['company-card__image-frame']}>
+          {profile.banner ? (
+            <img
+              src={profile.banner}
+              alt="Company Banner"
+              className={styles['company-card__image']}
+            />
+          ) : (
+            <img
+              className={styles['company-card__empty-image']}
+              src={`${process.env.REACT_APP_PUBLIC_URL}/svg/profile-view-image-empty.svg`}
+              alt={profile.name}
+            />
+          )}
+        </div>
+        <div className={styles['company-card__text-block']}>
+          <Tooltip
+            title={
+              profile.activities &&
+              profile.activities.map((element) => element.name).join(', ')
+            }
+            placement="bottom"
+            pointAtCenter={true}
+          >
+            <div className={styles['company-card__category-text']}>
+              {profile.activities &&
+                profile.activities
+                  .slice(0, 3)
+                  .map((activity) => activity.name)
+                  .join(' ')}
+            </div>
+          </Tooltip>
+          <div className={styles['company-card__text-block__header']}>
+            <div className={styles['company-card__name-text']}>
+              <Link
+                className={styles['company-card__name-text_link']}
+                to={`/profile-detail/${profile.id}`}
+              >
+                {profile.name}
+              </Link>
+              <br />
+            </div>
+          </div>
+          <Tooltip
+            title={profile.region}
+            placement="bottom"
+            pointAtCenter={true}
+          >
+            <div className={styles['company-card__region-text']}>
+              {profile.region
+                ? profile.region.length < 35
+                  ? `${profile.region}`
+                  : `${profile.region.substring(0, 35)}...`
+                : ''}
+            </div>
+          </Tooltip>
+          <div className={styles['company-card__footer_content']}>
+            <Tooltip
+              title={
+                profile.categories &&
+                profile.categories.map((element) => element.name).join(', ')
+              }
+              placement="bottom"
+              pointAtCenter={true}
+            >
+              <div className={styles['company-card__badges_block']}>
+                <CategoryBadges categories={profile.categories.slice(0, 3)} />
```

Comment:
Wondering why we pass only 3 items? Should we show the rest items withing the tooltip?

Reply:
We show all items in tooltip. Done